ContentPatcher/Framework/GenericModConfigMenuIntegrationForContentPack.cs

```diff
@@ -182,6 +193,19 @@ private void AddField(TConfigMenu menu, string name, ConfigField field)
                 formatAllowedValue: GetValueText
             );
         }
+
+        // image for preview image
+        if (field.PreviewImages?.Any() ?? false)
```

Comment:
This can just be a null check, as looping on an empty collection wont do anything